virtual_rainforest/models/plants/canopy.py

```diff
@@ -79,17 +83,21 @@ def build_canopy_arrays(
         n_canopy_layers: The maximum number of permitted canopy layers.
 
     Returns:
-        A tuple of two dimensional numpy arrays giving the canopy layer heights and leaf
-        area indices by cell id.
+        A tuple of two dimensional numpy arrays giving the canopy layer heights and then
+        leaf area indices, fapar and leaf mass by cell id.
```

Comment:
Am I right in thinking FAPAR is an acronym?

```suggestion
        leaf area indices, FAPAR and leaf mass by cell id.
```

Reply:
Hah - yes it is. It is usually written as fAPAR because it is the fraction of APAR ($f_{APAR}$).